lib/screens/network_settings_screen.dart

```diff
@@ -44,45 +38,31 @@ class _NetworkSettingsScreenState extends State<NetworkSettingsScreen> {
           Divider(),
           ActiveNetworkDisplay(),
           PublicAddressSelector(),
-          if (Platform.isAndroid || Platform.isIOS)
-            ListTile(
-              leading: Icon(Icons.info_outline),
-              subtitle: Text(AppLocalizations.of(context)!.preferHomeNetworkInfoBox),
-            ),
           HomeNetworkSelector(),
-          HomeNetworkNameSelector(),
-          TextButton(
-              onPressed: () async {
-                bool allowed = await ensureLocationPermissions(context);
-                if (!allowed) return;
-
-                // Get the current network name
-                String? network = await NetworkInfo().getWifiName();
-                if (network == null) {
-                  GlobalSnackbar.message((context) => AppLocalizations.of(context)!.preferHomeNetworkFailedToReadNetworkNameError);
-                  return;
-                }
-
-                // android returns the network name with quotes
-                GetIt
-                  .instance<FinampUserHelper>()
-                  .currentUser
-                  ?.update(
-                    newHomeNetworkName: network.replaceAll("\"", ""));
-                await changeTargetUrl();
-              },
-              child: Text(AppLocalizations.of(context)!.preferHomeNetworkGetNetworkNameButton)),
           HomeNetworkAddressSelector(),
           TextButton(
             onPressed: () async {
-              await GetIt.instance<JellyfinApiHelper>().pingServer()
+              final user = GetIt.instance<FinampUserHelper>().currentUser!;
+              final active = user.baseURL;
+              final local = user.homeAddress;
+              await GetIt.instance<JellyfinApiHelper>().pingActiveServer()
                 .then((available) {
                   if (available) {
-                    GlobalSnackbar.message((context) => AppLocalizations.of(context)!.pingSuccessful);
+                    GlobalSnackbar.message((context) => AppLocalizations.of(context)!.pingSuccessful(active));
                   } else {
-                    GlobalSnackbar.message((context) => AppLocalizations.of(context)!.pingFailed);
+                    GlobalSnackbar.message((context) => AppLocalizations.of(context)!.pingFailed(active));
                   }
                 });
+              if (active != local) {
```

Comment:
So we're pinging twice now? Maybe have a single, more customized message for the result? Something like "Both addresses valid", "Local address invalid", "Remote address invalid"?